e2e/tests/mcp-setup-wizard.spec.ts

```
@@ -0,0 +1,330 @@
+      await spaNavigate(page, '/kuadrant/mcp/overview');
+
+      await expect(page.getByRole('heading', { name: 'MCP Management' })).toBeVisible({
+        timeout: 15_000,
+      });
+      await expect(page.getByRole('heading', { name: 'Get started' })).toBeVisible();
+    });
+
+    test('shows setup wizard button', { tag: '@smoke' }, async ({ page }) => {
+      await spaNavigate(page, '/kuadrant/mcp/overview');
+
+      await expect(page.locator('[data-test="mcp-setup-wizard-button"]')).toBeVisible({ timeout: 15_000 });
```

ESLint

[error] Replace `·timeout:·15_000` with `⏎········timeout:·15_000,⏎·····`

(prettier/prettier)